### PR TITLE
fixed issues with overview page rendering

### LINKED ISSUE #381: Overview page rendering
Overview page is always rendered from other pages especially on interfaces like terminal, fyne and nuklear rendering overview page when application is launched.

terminal ui
image

### PR DESCRIPTION
**Pull request by Sirmorrison**
fixed #381 

fixed issues with overview page always rendering from other pages on interfaces rendering overview page on launch like terminal and fyne.

### CODE REVIEW

File: fyne/sync.go [hunk 1]
```diff
@@ -47,13 +47,19 @@ func (app *fyneApp) showSyncWindow() {
...
 		progressReport := report.Read()
 
 		progressBar.SetValue(float64(progressReport.TotalSyncProgress))
 
 		if progressReport.Status == defaultsynclistener.SyncStatusSuccess {
-			app.loadMainWindowContent()
+			if !syncCompleted {
+				app.loadMainWindowContent()
+				syncCompleted = true
+			}else{
```
Comment (ademuanthony) [inline fyne/sync.go]: This else block can be removed

File: terminal/pages/sync.go [hunk 2]
```diff
@@ -89,11 +89,18 @@ func LaunchSyncPage(tviewApp *tview.Application, walletMiddleware app.WalletMidd
...
+	var syncCompleted bool
 	walletMiddleware.SyncBlockChain(false, func(report *defaultsynclistener.ProgressReport) {
 		progressReport := report.Read()
 
 		if progressReport.Status == defaultsynclistener.SyncStatusSuccess {
-			afterSyncing()
+			if !syncCompleted {
+				afterSyncing()
+				syncCompleted = true
+			}else{
```
Comment (ademuanthony) [inline terminal/pages/sync.go]: this `else` block can be removed

File: terminal/pages/sync.go [hunk 3]
```diff
@@ -89,11 +89,18 @@ func LaunchSyncPage(tviewApp *tview.Application, walletMiddleware app.WalletMidd
 }
 
 func startSync(walletMiddleware app.WalletMiddleware, updateStatus func([]string), handleError func(string), afterSyncing func()) {
+	var syncCompleted bool
```
Comment (ademuanthony) [inline terminal/pages/sync.go]: I guess this variable is tracking whether the main page has been loaded or not. Please rename

File: fyne/sync.go [hunk 4]
```diff
@@ -47,13 +47,19 @@ func (app *fyneApp) showSyncWindow() {
 	app.resizeAndCenterMainWindow(syncWindowContent)
 	app.mainWindow.Show()
 
+	var syncCompleted bool
```
Comment (ademuanthony) [inline fyne/sync.go]: I guess this variable is tracking whether the main page has been loaded or not. Please rename

**Review by ademuanthony — CHANGES_REQUESTED** [review]

File: fyne/sync.go (hunk 4 above)
Comment (Sirmorrison) [reply fyne/sync.go]: Its tracking is sync was completed. Reason is after sync is completed the process starts again in background.

**Review by ademuanthony — APPROVED** [review]

**Pull request merged**